index.js

```diff
@@ -830,13 +838,14 @@
                       reprs.join(_(', ')));
 
     var padding = _(showTypeSig_(expTypes.slice(0, index)));
+    var f = underline(typePath[0])(propPath);
 
     return new TypeError(unlines([
```

Comment:
I know I already said it, but I love the formatting of these now with the longer lines + `unlines` :)